{"name":"Logintest","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.NullPointerException","trace":"java.lang.NullPointerException\r\n\tat Acu.ResumeTracker.TestClass.Resume_LoginTest_1.Logintest(Resume_LoginTest_1.java:35)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(Unknown Source)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:764)\r\n\tat org.testng.TestRunner.run(TestRunner.java:585)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1140)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1069)\r\n\tat org.testng.TestNG.run(TestNG.java:1037)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"saveFailureScreenShot","source":"a993c216-cf1e-471b-8b33-deba0f9d8c87-attachment"},{"name":"Logintest failed and screenshot taken!","source":"10cb2aad-c2da-4564-b820-a5477c314ab5-attachment","type":"text/plain"}],"parameters":[],"start":1605255901332,"stop":1605255902842,"uuid":"e9cf26ef-c8cc-4451-a0d5-a13404445b1f","historyId":"5552c8f635b7c87e075fb3711e5a1d85","fullName":"Acu.ResumeTracker.TestClass.Resume_LoginTest_1.Logintest","labels":[{"name":"package","value":"Acu.ResumeTracker.TestClass.Resume_LoginTest_1"},{"name":"testClass","value":"Acu.ResumeTracker.TestClass.Resume_LoginTest_1"},{"name":"testMethod","value":"Logintest"},{"name":"parentSuite","value":"AcurusResume"},{"name":"suite","value":"ResumeTracker"},{"name":"subSuite","value":"Acu.ResumeTracker.TestClass.Resume_LoginTest_1"},{"name":"host","value":"DESKTOP-HR5OPSO"},{"name":"thread","value":"6428@DESKTOP-HR5OPSO.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}